.github/workflows/mobilecoin-dev-cd.yaml

```diff
@@ -380,24 +380,18 @@ jobs:
 # Reset existing namespace
 #################################
   dev-reset:
-    runs-on: [self-hosted, Linux, small]
     needs:
     - generate-metadata
-    steps:
```

Comment:
Convert this from a workflow_dispatch to the workflow_call. 

This is a little bit more efficient since you don't need to consume a runner to wait on the dispatch to complete.

GHA didn't support retries on workflow_calls when we were first developing these patterns, I'll be converting the rest as we go.